### ISSUE TITLE
Non-Functional Reply Panel Text and Message Icon in Rocket.Chat UI on Browser in PC

### ISSUE THREAD

**Issue by Legend101Zz** [RocketChat/Rocket.Chat#28166]
### Description:

Currently, in Rocket. Chat's PC application, in order to view the message reply panel, users must click on the reply button. However, the replies text and message icon that appears with it is non-functional, providing a confusing and bothersome user experience. To improve the user experience and eliminate confusion, the non-functional replies text and message icon should be removed or made functional.

### Steps to reproduce:

1. Open any Rocket Chat Channel in a web browser on a PC.
2. Clicking on the replies text or the message icon does not open the message reply panel.

### Expected behavior:

The message reply panel should be displayed upon clicking 

### Actual behavior:

<img width="1149" alt="Screenshot 2023-02-25 at 00 35 18" src="https://user-images.githubusercontent.com/96632943/221297863-51289608-c3f6-46f6-b615-3cd102717b9e.png">

The message reply panel does not open on clicking the replies text or the message icon 

### Server Setup Information:

- Version of Rocket.Chat Server: 
- Operating System: 
- Deployment Method: 
- Number of Running Instances: 
- DB Replicaset Oplog: 
- NodeJS Version: 
- MongoDB Version:

**Comment by XshubhamX** [comment]
@Legend101Zz  According to me, it's not a problem, more of a means to learn more about the thread. There are 4 messages in this thread, according to the message symbol and the text next to it, and I believe this makes sense. @dudanogueira  Your insights will be helpful in this situation since the internal team has to be informed if any UI components change.

**Comment by Legend101Zz** [comment]
@XshubhamX Thanks for the valuable input. It was just a thing I felt when I used Rocket Chat for the first time, as I had a hard time figuring out how to see replies to a particular message, what's your input on that ?

**Comment by fthommen** [comment]
I strongly agree with @Legend101Zz . It's also irritating, as the "normal" - and IMHO to be expected - user experience with all other chat systems which I know, is different in that respect: Clicking on the thread information opens the thread itself.

Additionally, the text of the "Reply" button is basically wrong, as its primary function is to open the respective thread, not to write a reply. For that there is the "Reply in Thread" button on the upper right edge of each post
